csv_generator/process_xml_zip.py

```diff
@@ -66,6 +69,11 @@ def main():
     parser.add_argument('--source-zip', type=str, help='Source zip file e.g. some-file.zip')
     parser.add_argument('--output-dir', default=DEFAULT_OUTPUT_DIR,
                         type=str, help='CSV output directory. Defaults to {}'.format(DEFAULT_OUTPUT_DIR))
+    parser.add_argument(
+        '--batch',
```

Comment:
Style: I have to admit I prefer starting on the next line and indenting it always the same rather than aligning it to the parameters of the first line which seem to waste space and would have to change when renaming functions. I also prefer putting the closing bracket at the end to make it easier for the eye to see where the end of that block is and the next line should start. (Probably a topic for another day)

Reply:
Both are valid under [pep8](https://www.python.org/dev/peps/pep-0008/#indentation) and just come down to preference. If I am starting a project I will use my preference, if I am contributing to one I will follow whats been used to keep it consistent, as consistency across a project is more important for the next reader.